### DISCUSSION TITLE
How to use images in app?

### DISCUSSION THREAD

**Discussion by nett00n** [orgzly-revived/orgzly-android-revived#225] [Q&A] [answered]
There is a option to use images in notes in settings
![image](https://github.com/orgzly-revived/orgzly-android-revived/assets/1846116/df9ab3f6-5553-4ca5-93f6-242214bfd772)

But i cannot find, how to attach them inside task:
![image](https://github.com/orgzly-revived/orgzly-android-revived/assets/1846116/6ef679e8-0a4f-49bc-9b1a-c1e3e511d062)
There is no such option in hamburger menu, no `image` field in the task and no option in long-tap pop-up in `description`

**Comment by cbkerr** [comment, marked as answer]
Manually type`[[file:path]]` where `path` is the path to the file, or relative to the orgzy directory. I agree it would be nice to have a way to insert this into the notebook, perhaps on long press as you suggest.

There is some work on making it easier to add images from other sources: https://github.com/orgzly-revived/orgzly-android-revived/pull/130
